web_common/web_common/styles/main.scss

```diff
@@ -233,7 +233,7 @@ a {
     }
 
     tr {
-        &:nth-child(even) {
+        &:nth-type-of(even) {
```

Comment:
I looked to see if nth-child was used anywhere else. I think you should just duplicate this.

Reply:
Although maybe if you have both, it will be the wrong behavior. I don't know enough about CSS.

Reply:
If we duplicate it then it will be wrong because, for example, the 3rd visible thing but 4th overall will be gray. These rules are only applied `tr` elements inside of an element whose class is `data-table`. I do not think we ever have anything but `tr` elements inside the `tbody` of `table` whose class is `data-table` (except as added by this PR).